modules/get-started/pages/quick-start.adoc

```diff
@@ -226,75 +430,235 @@ include::get-started:partial$install-rpk-homebrew.adoc[]
+
+NOTE: For this quickstart, Redpanda uses MinIO as the object storage provider. MinIO is not supported for production deployments. For production, see xref:deploy:deployment-option/self-hosted/manual/production/requirements.adoc#object-storage-providers-for-tiered-storage[Object storage providers for Tiered Storage].
+
+. Check the size of remote and local disks:
++
+[,bash]
+----
+rpk cluster logdirs describe --topics logins
+----
++
+[.no-copy]
+----
+BROKER  DIR                     TOPIC   PARTITION  SIZE     ERROR
+1       /var/lib/redpanda/data  logins  0          3364032
+1       remote://redpanda       logins  0          3358193
+----
++
+- Local storage: `/var/lib/redpanda/data` holds recent data.
+- Remote storage: `remote://redpanda` holds offloaded data.
+
+. Open MinIO at http://localhost:9001/browser to view your data stored in the S3-compatible object store.
++
+Login credentials:
++
+- Username:
++
+```
+minio
 ```
+- Password:
 +
-NOTE: The Redpanda broker returns a list of all broker addresses, so `rpk` can communicate with all brokers directly. Each broker returns its configured `advertise-*` address that matches the port to which the initial connection has been made.
+```
+redpandaTieredStorage7
+```
+
+See also: xref:manage:tiered-storage.adoc[]
+
+// (step {"action":"runShell", "command": "docker stop redpanda-2", "workingDirectory": "../docker-compose/three-brokers"})
+// (step {"action":"wait"})
+// (step {"action":"goTo", "url": "http://localhost:8080/overview"})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "degraded-cluster.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+// (step {"action":"runShell", "command": "docker start redpanda-2", "workingDirectory": "../docker-compose/three-brokers"})
 
-== Explore your topic
+== Deploy a data streaming pipeline
 
-To explore your topic, you can use Redpanda Console, which you deployed in the `docker-compose.yml` file and is running locally on port 8080.
-Redpanda Console is a developer-friendly web UI for managing and debugging your Redpanda cluster and your applications.
+Deploying a data streaming pipeline in Redpanda involves configuring and running connectors that ingest, transform, and route data between various sources and sinks. Redpanda Connect simplifies this process by allowing you to manage these pipelines using declarative YAML configuration files and the `rpk` CLI.
 
-. http://127.0.0.1:8080/overview[Open Redpanda Console^] in a web browser.
+. Deploy the pipeline:
 +
-An overview of the cluster status, cluster health, and broker details is displayed.
+[,bash]
+----
+rpk connect run ../generate-profiles.yaml
+----
++
+NOTE: The pipeline configuration is in the `docker-compose/generate-profiles.yaml` file that you downloaded. This file is commented to explain how it works.
```

Comment:
I had to Ctrl-C here. I think this needs more explanation + link directly to `generate-profiles.yaml`.